packages/commerce-sdk-react/src/components/StorefrontPreview/storefront-preview.tsx

```diff
@@ -8,11 +8,12 @@
 import React, {useEffect} from 'react'
 import PropTypes from 'prop-types'
 import {Helmet} from 'react-helmet'
-import {detectStorefrontPreview, getClientScript} from './utils'
+import {CustomPropTypes, detectStorefrontPreview, getClientScript, noop} from './utils'
 import {useHistory} from 'react-router-dom'
 import type {LocationDescriptor} from 'history'
 
 type GetToken = () => string | undefined | Promise<string | undefined>
+type ContextChangeHandler = () => undefined | Promise<undefined>
```

Comment:
I've been thinking about what if anything we could return from this function. But I don't see a real reason to return anything at this moment. If there is an error you are expected to throw and it will get caught internally.

Reply:
```suggestion
type ContextChangeHandler = () => void | Promise<void>
```
`void` is the standard return type to use for "ignore the return value".